packages/core/src/components/EmptyState/EmptyState.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import { Button } from '@material-ui/core';
```

Comment:
This should be moved down to the rest of the imports (copyright header at the top).